storage/run_kvrocks.sh

```diff
@@ -3,4 +3,4 @@
 set -e
 set -x
 
-../../kvrocks/build/kvrocks -c kvrocks.conf
+../kvrocks/build/kvrocks -c kvrocks.conf
```

Comment:
Same as redis, this is going to break existing installs.

Reply:
I reverted, it's not working for me now ;-) We can maybe have a solution which is working in both cases. Or I simply move my ``kvrocks`` and ``redis`` folders inside new folders.

Reply:
the install expects redis and kvrocks outside of the repository: https://github.com/cve-search/vulnerability-lookup/blob/main/INSTALL.md#prerequisites